experimental/sql/tools/rel_opt.py

```diff
@@ -8,6 +8,7 @@
 
 from src.ibis_frontend import ibis_to_xdsl
 from dialects.ibis_dialect import Ibis
+from dialects.rel_tree import RelationalTree
```

Comment:
Another possible name is `rel_alg` as reference to the mathematical meaning of relational algebra, which is just about expressions. The SSA form could be called `rel_prog` or `rel_ssa` because it is a "program" that works on SSA values that are relations...

Reply:
I really like that, since I feel that we really are modeling relational algebra in this dialect, while the dialect one lower is already a kind of `implementation`.